README.md

```diff
@@ -92,15 +92,45 @@ Here are our six core certifications:
   <br />
   **Projects**: Timestamp Microservice, Request Header Parser, URL Shortener, Exercise Tracker, File Metadata Microservice
 
-#### 6. Information Security and Quality Assurance Certification
+#### 6. Quality Assurance Certification
 
-- [Information Security with HelmetJS](https://learn.freecodecamp.org/information-security-and-quality-assurance/information-security-with-helmetjs)
 - [Quality Assurance and Testing with Chai](https://learn.freecodecamp.org/information-security-and-quality-assurance/quality-assurance-and-testing-with-chai)
 - [Advanced Node and Express](https://learn.freecodecamp.org/information-security-and-quality-assurance/advanced-node-and-express)
   <br />
   <br />
-  **Projects**: Metric-Imperial Converter, Issue Tracker, Personal Library, Stock Price Checker, Anonymous Message Board
+  **Projects**: Metric-Imperial Converter, Issue Tracker, Personal Library, Sudoku Solver, American British Translator
+   
+#### 7. Scientific Computing with Python Certification
 
+- [Introduction to Python for Everybody](https://learn.freecodecamp.org/scientific-computing-with-python/python-for-everybody)
+  <br />
+  <br />
+  **Projects**: Arithmetic Formatter, Time Calculator, Budget App, Polygon Area Calculator, Probability Calculator
+
+#### 8. Data Analysis with Python Certification
+
+- [Data Analysis with Python Course](https://learn.freecodecamp.org/data-analysis-with-python/data-analysis-with-python-course)
+- [NumPy](https://learn.freecodecamp.org/data-analysis-with-python/numpy)
+  <br />
+  <br />
+  **Projects**: Mean-Variance-Standard Deviation Calculator, Demographic Data Analyzer, Medical Data Visualizer, Page View Time Series Visualizer, Sea Level Predictor
+  
+#### 9. Information Security Certification
+
+- [Information Security with HelmetJS](https://learn.freecodecamp.org/information-security/information-security-with-helmetjs)
+- [Python for Penetration Testing](https://learn.freecodecamp.org/information-security/python-for-penetration-testing)
+  <br />
+  <br />
+  **Projects**: Stock Price Checker, Anonymous Message Board, Port Scanner, SHA-1 Password Cracker, Secure Real Time Multiplayer Game 
+  
+#### 10. Machine Learning with Python Certification
+
+- [TensorFlow](https://learn.freecodecamp.org/machine-learning-with-python/tensorflow)
+- [How Neural Networks Work](https://learn.freecodecamp.org/machine-learning-with-python/how-neural-networks-work)
+  <br />
+  <br />
+  **Projects**: Rock Paper Scissors, Cat and Dog Image Classifier, Book Recommendation Engine using KNN, Linear Regression Health Costs Calculator, Neural Network SMS Text Classifier
+  
 #### Full Stack Development Certification
 
 Once you have earned all 6 of these certifications, you'll be able to claim your freeCodeCamp.org Full Stack Development Certification. This final distinction signifies that you’ve completed around 1,800 hours of coding with a wide range of web development tools.
```

Comment:
If we are updating the certifications, the Full Stack certification could use some clarification as well.

Reply:
I have committed those changes, I'm not sure how does that get updated on the PR.

Reply:
When you commit the changes to your branch, it automatically updates the PR as well. 🙂

Reply:
Could you please check out the new changes made as requested?